client/src/index.js

```diff
@@ -9,14 +9,14 @@ import { UserContextProvider } from 'Contexts/UserContext';
 import { RoomContextProvider } from 'Contexts/RoomContext';
 
 ReactDOM.render(
-  <React.StrictMode>
-    <RoomContextProvider>
-      <UserContextProvider>
-        <SocketIOProvider url="/">
-          <App />
-        </SocketIOProvider>
-      </UserContextProvider>
-    </RoomContextProvider>
-  </React.StrictMode>,
+  // <React.StrictMode>

```

Comment:
Można to usunąć z czystym sumieniem, gryzie się to z Mui a nie wnosi nic do projektu poza błędami których i tak już mamy pod dostatkiem